py/runai_model_streamer/runai_model_streamer/safetensors_streamer/safetensors_pytorch.py

```diff
@@ -52,22 +52,24 @@ def __init__(self, blob: any, offset: int) -> None:
                 self.read_sizes.append(self.tensors_metadata[i].get_bytesize())
 
     @staticmethod
-    def from_files(fs: FileStreamer, filenames: str) -> List[SafetensorsMetadata]:
-        fs.stream_files([FileChunks(filename, 0, [SAFETENSORS_HEADER_BUFFER_SIZE]) for filename in filenames])
+    def from_files(fs: FileStreamer, filenames: List[str], s3_credentials: Optional[S3Credentials], device_str: str) -> List[SafetensorsMetadata]:
```

Comment:
Why do you need the "device_str" parameter in the "from_file" function? 
It reading the files metadata, its always to the CPU